pdns/recursordist/RECURSOR-MIB.txt

```diff
@@ -1032,6 +1035,21 @@ udp6InCsumErrors OBJECT-TYPE
         "Number of UDP6 in checksum errors (Linux only)"
     ::= { stats 123 }
 
+unauthorizedNotify OBJECT-TYPE
+    SYNTAX Counter64
+    MAX-ACCESS read-only
+    STATUS current
+    DESCRIPTION
+        "Number of unauthorized NOTIFY operations"
+    ::= { stats 124 }
+
+disallowedNotify OBJECT-TYPE
+    SYNTAX Counter64
+    MAX-ACCESS read-only
+    STATUS current
+    DESCRIPTION
+        "Number of disallowed NOTIFY operations"
```

Comment:
please mention if this about the domain or IP

Reply:
Done, but this points out a potential issue. The two new configuration settings are `allow-notify-for` and `allow-notify-from`, but the metrics use `disallowed` and 'unauthorized` for those two cases. This is not consistent.

Possibly the metrics could be named "sourceDisallowedNotify" and "zoneDisallowedNotify".

Reply:
Yes, I would very much like metric names that reflect the settings names.

Reply:
Will do.